server/service.go

```diff
@@ -610,9 +665,9 @@ func (m *BoostService) processCapellaPayload(w http.ResponseWriter, req *http.Re
 	requestCtx, requestCtxCancel := context.WithCancel(context.Background())
 	defer requestCtxCancel()
 
-	for _, relay := range relays {
+	for _, r := range relays {
```

Comment:
Guessing this change was made because the inner function used the same name? Good change.

Reply:
`relay` collides with the package name, hence the change.

Reply:
Ah gotcha. Makes sense.